tests/fixtures/preload.ts

```diff
@@ -17,11 +17,14 @@ async function toMatchGerberSnapshot(
 
   try {
     const stackup = await pcbStackup(layers)
-
+    const gerbersArray: string[] = [];
     // We'll use the top layer SVG for comparison, but you could choose bottom or both
-    const svg = stackup.top.svg
-
-    return expect(svg).toMatchSvgSnapshot(testPathOriginal, svgName)
+    for (const item of Object.keys(stackup)) {
+        if (stackup[item].svg) {
+          gerbersArray.push(expect(stackup[item].svg).toMatchSvgSnapshot(import.meta.path, svgName + `${item}`))
+        }
+    }
+      return gerbersArray;
```

Comment:
this method has to return a `MatchOutput` or something for it to be compliant with bun, you'll need to combine them (types will be useful here)